Makefile

```diff
@@ -98,6 +98,12 @@ test: generate fmt vet manifests bin/setup-envtest
 		-timeout 1h
 	cd properties && go test -coverprofile cover.out -cover -failfast -v -covermode=count ./pkg/... ./internal/...
 
+# Run e2e tests
+test-e2e:
+	go test github.com/banzaicloud/koperator/tests/e2e \
+		-v \
+		-timeout 5m
+
```

Comment:
Maybe we can add more verbose output for ginkgo:
--ginkgo.show-node-events \
		--ginkgo.trace \
		--ginkgo.v

Reply:
Why do you think so?
In case of failure we can rerun the automated tests with more verbose output, but I found the manually added logging adequate enough, especially from the Ginkgo side, especially if we want to order the test specifications (which I'm debating in myself still which would be the best).

Reply:
I agree with you, maybe it is unecessary but what is the advantage to not run with more verbose output? 
Maybe it is better to not re-run a 15 min test again to get more verbose output when CI pipeline fail. What if the error does not happen every time?

Reply:
Fixed in 2ca83fe.